aloha-core/src/main/scala/com/eharmony/aloha/io/fs/package.scala

```diff
@@ -33,6 +33,7 @@ package object fs {
         def inputStream: InputStream
         def outputStream(append: Boolean = false): OutputStream
         def localFile: Option[File]
+        def isLocal = localFile.isDefined
```

Comment:
Sure. That's a little better.  It took me a second to figure out why, but I agree.  That actually checks foe the file's existence.



On Aug 21, 2015, 4:29 PM, at 4:29 PM, jmorra <notifications@github.com> wrote:
>> @@ -33,6 +33,7 @@ package object fs {
>>          def inputStream: InputStream
>>          def outputStream(append: Boolean = false): OutputStream
>>          def localFile: Option[File]
>> +        def isLocal = localFile.isDefined
>
>Should this say
>{code}
>localFile.map(_.isFile).getOrElse(false)
>{code}
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/eHarmony/aloha/pull/35/files#r37687276